src/app/new-auth/forgot-password/forgot-password.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="wrapper">
+    <div [ngClass]="{'failed': showError}" class="content-box">
+        <clark-error-banner
+            [errorMessage]="errorMessage"
+            class="errBanner"
+        ></clark-error-banner>
+        <img src="../../../assets/images/logo.png" alt="CLARK Logo" />
+
+        <div *ngIf="!showDone" [@switchViews]="showDone">
+            <form (ngSubmit)="submit()">
+                <h1>Request Password Reset</h1>
+            
+            <clark-input-field
+                phold="Email"
+                [control]="emails.get('email')"
+                #email
+                ngModel
+                (input)="toggleButton()"
+                name="email"
+                [errorMsg]="emailErrorMsg"
+            >
+            </clark-input-field>
+
+            <clark-input-field
+                phold="Confirm Email"
+                [control]="emails.get('confirmEmail')"
+                confirmType="Email"
+                ngModel
+                (input)="toggleButton()"
+                name="confirmEmail"
+            >
+            </clark-input-field>
+
+            <button (click)="submit()" [ngClass]="submitButton" [disabled]="submitButton === 'disabled'" [@toggleButton]="submitButton">Request Password Reset</button>
+            </form>
+            
+        </div>
+
+        <div *ngIf="showDone" class="done" [@switchViews]="showDone">
+            <h2>Email is on the way!</h2>
+            <p>Check your inbox and spam folders</p>
+            <button routerLink="../login">Login</button>
```

Comment:
Should the user be redirected to login or back home?

Reply:
The button will be removed. The workflow should be that the user shouldn't go anywhere within CLARK after that email is sent. The intent is that the user will go to their email and reset their password.